lib/bloc/bloc_provider.dart

```diff
@@ -25,13 +25,13 @@ class BlocProvider<T extends BlocBase> extends StatefulWidget {
 
 class _BlocProviderState<T> extends State<BlocProvider<BlocBase>> {
   @override
-  void dispose() {
-    widget.bloc.dispose();
-    super.dispose();
+  Widget build(BuildContext context) {
```

Comment:
Is this just a format change? If yes, then can you please revert it?

Reply:
ok, no probs